dataland-frontend/tests/e2e/fixtures/sfdr/SfdrPreparedFixtures.ts

```diff
@@ -40,18 +43,70 @@ export function generateSfdrPreparedFixtures(): Array<FixtureData<SfdrData>> {
  */
 function manipulateFixtureForOneSfdr(input: FixtureData<SfdrData>): FixtureData<SfdrData> {
   input.companyInformation.companyName = "company-with-one-sfdr-data-set";
-  input.t.social!.general!.annualReport = "/test_pdf.pdf";
+  input.t.general.general.referencedReports = undefined;
   return input;
 }
 
+/**
+ * Sets the company name and the date in the fixture data to a specific string
+ * @param input Fixture data to be manipulated
+ * @returns the manipulated fixture data
+ */
+function manipulateFixtureForOneFilledSubcategory(input: FixtureData<SfdrData>): FixtureData<SfdrData> {
+  input.companyInformation.companyName = "CompanyInformationWithSfdrData";
```

Comment:
The company name of our prepared fixtures usually holds a specific description of the prepared fixture.
So I suggest choosing smth. like "company-with-one-filled-sfdr-subcategory" here.

Reply:
Renmaed to companyWithOneFilledSfdrSubcategory

Reply:
👍